sqlglot/dialects/sqlite.py

```diff
@@ -92,6 +92,7 @@ class Generator(generator.Generator):
         NVL2_SUPPORTED = False
         JSON_PATH_BRACKETED_KEY_SUPPORTED = False
         SUPPORTS_CREATE_TABLE_LIKE = False
+        SUPPORTS_TABLE_ALIAS_COLUMNS = False
```

Comment:
Let's test this, i.e. show how transpiling something like `SELECT * FROM t(c)` to SQLite (a) generates `SELECT * FROM t` and (b) logs a warning (look for the `self.assertLogs` tests to see how (b) is implemented).

Reply:
Ack

Reply:
Done, introduced that test case to `test_sqlite.py` in accordance to similar checks in `test_bigquery.py`